example-widget-mui/src/ThemePage/ThemePage.tsx

```diff
@@ -774,6 +776,96 @@ export function WidgetListItem({
   );
 }
 
+export function AvatarsDemo() {
+  return (
+    <Box m={1}>
+      <Typography variant="h2">Avatars</Typography>
+      <p>Avatars can be used together with users or rooms.</p>
+      <p>
+        <ElementAvatar userId="@user:matrix.org" /> Avatar without image and
+        name
+      </p>
+      <p>
+        <ElementAvatar userId="@user:matrix.org" displayName="Display Name" />
+        Avatar without image
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@user:matrix.org"
+          displayName="Display Name"
+          avatarUrl="https://example.com/test.png"
+        />
+        Avatar with broken image URL
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@user:matrix.org"
+          displayName="Display Name"
+          avatarUrl="https://example.com/test.png"
+        />
+        Avatar with broken image URL
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="mxc://matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+        />
+        Avatar with mxc: URL (might not resolve)
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+        />
+        Avatar
+      </p>
+      <p>
+        <ElementAvatar
+          userId="@oliver.sand:matrix.org"
+          displayName="Oliver Sand"
+          avatarUrl="https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/KbhRFOIJekHQpMVIZGpuNxBG"
+          sx={(theme) => ({
+            outline: `${theme.palette.primary.main} solid 2px`,
+            border: `2px solid ${theme.palette.background.default}`,
+          })}
+        />
+        Avatar with outline
```

Comment:
I wonder how well this works if the letter is displayed. The font size might be too big in that case?

<img width="157" alt="image" src="https://user-images.githubusercontent.com/720821/223361546-00dafefb-d1ab-40ce-bf49-84cee6d6f1aa.png">

Reply:
Hmm, both border and outline are outside and should not affect the content 🤔 What do you mean exactly?

Reply:
The image shrinks when the border is applied:

<img width="331" alt="image" src="https://user-images.githubusercontent.com/720821/223362447-4f1c3f9d-f69b-4003-903f-5cb0d5a46bd8.png">

<img width="331" alt="image" src="https://user-images.githubusercontent.com/720821/223362493-9108c4d6-bb59-4198-8bda-84fa493056f7.png">

But the text stays the same size and feels slightly too large in my opinion:

<img width="52" alt="image" src="https://user-images.githubusercontent.com/720821/223362626-6fbc20bd-4daf-4a49-8359-b101633fe670.png">

<img width="63" alt="image" src="https://user-images.githubusercontent.com/720821/223362654-cb7df32c-363b-40b0-a8a5-008c33366a98.png">